pkg/ocicni/ocicni.go

```diff
@@ -278,13 +278,13 @@ func loadNetworks(confDir string, cni *libcni.CNIConfig) (map[string]*cniNetwork
 		if strings.HasSuffix(confFile, ".conflist") {
 			confList, err = libcni.ConfListFromFile(confFile)
 			if err != nil {
-				logrus.Warningf("Error loading CNI config list file %s: %v", confFile, err)
+				logrus.Errorf("Error loading CNI config list file %s: %v", confFile, err)
 				continue
 			}
 		} else {
 			conf, err := libcni.ConfFromFile(confFile)
 			if err != nil {
-				logrus.Warningf("Error loading CNI config file %s: %v", confFile, err)
+				logrus.Errorf("Error loading CNI config file %s: %v", confFile, err)
```

Comment:
Same question here about the change from "Warning" to "Error".

Reply:
fixed!

Reply:
Same here.